src/job.rs

```diff
@@ -1,10 +1,24 @@
-extern chrono;
+extern crate chrono;
 
 use chrono::prelude::*;
 
-mod job {
-    struct Job<T> {
-        activity: Fn<T>,
-        deadline: DateTime,
+#[derive(Debug)]
+struct Activity {
```

Comment:
Can probably be renamed, that or could be a sum type: it could be a command+arg pair or it could be a function closure?